scripts/mocha.sh

```diff
@@ -2,7 +2,11 @@
 
 set -e -x
 
-mocha \
-  -r should \
-  --compilers coffee:coffee-script/register,js:babel-core/register \
-  -t 30000
+for file in "$@"
+  do
+    mocha \
```

Comment:
This should use `node_modules/.bin/mocha` so it doesn't depend on the global mocha install (this is implicit in package.json `scripts`).